platform/requirements/requirements.txt

```diff
@@ -1,11 +1,11 @@
 # Common libraries and basic python add-ons
```

Comment:
Perhaps move platform builder to a common directory later?

Reply:
That would be great.